fractal-puppy/Engine
# Added gameplay to Graveyard scene

To test:
Open the "Level0-TheGraveyard" scene and test that the player moves correctly, the enemies can be killed and they can kill you. Also, check that the player life mask decreases as you take damage and the numbers with the damage done appear on the screen.
